docs/architecture/airbyte-specification.md

```diff
@@ -0,0 +1,180 @@
+# The Airbyte Specification
+
+## Summary
+This page will describe the Airbyte Specification.
```

Comment:
We should have a good short blurb that makes it clear what is being provided here and that the spec is:
- docker based
- a CLI
- provides structured stdout message passing
- defines sources / targets / messages
- is designed to work with Airbyte but is an independent standard.

Reply:
thanks this is helpful